File: snippets/typescript/search_point_groups.ts
import { QdrantClient } from "@qdrant/js-client-rest";

const client = new QdrantClient({ host: "localhost", port: 6333 });

client.searchPointGroups("{collection_name}", {
    vector: [1.1],
    group_by: "document_id",
    limit: 4,
    group_size: 2,
});

File: snippets/csharp/search_point_groups.cs
using Qdrant.Client;

var client = new QdrantClient("localhost", 6334);

await client.SearchGroupsAsync(
  collectionName: "{collection_name}",
  vector: new float[] { 1.1f },
  groupBy: "document_id",
  limit: 4,
  groupSize: 2
);
